docs/api/text/freqdist.rst

```diff
@@ -5,74 +5,119 @@ Token Frequency Distribution
 
 A method for visualizing the frequency of tokens within and across corpora is frequency distribution. A frequency distribution tells us the frequency of each vocabulary item in the text. In general, it could count any kind of observable event. It is a distribution because it tells us how the total number of word tokens in the text are distributed across the vocabulary items.
 
-.. code:: python
+Note that the ``FreqDistVisualizer`` does not perform any normalization or vectorization, and it expects text that has already be count vectorized.
 
-    from yellowbrick.text import FreqDistVisualizer
-    from sklearn.feature_extraction.text import CountVectorizer
+We first instantiate a ``FreqDistVisualizer`` object, and then call ``fit()`` on that object with the count vectorized documents and the features (i.e. the words from the corpus), which computes the frequency distribution. The visualizer then plots a bar chart of the top 50 most frequent terms in the corpus, with the terms listed along the x-axis and frequency counts depicted at y-axis values. As with other Yellowbrick visualizers, when the user invokes ``poof()``, the finalized visualization is shown.  Note that in this plot and in the subsequent one, we can orient our plot vertically by passing in ``orient='v'`` on instantiation (the plot will orient horizontally by default):
 
-Note that the ``FreqDistVisualizer`` does not perform any normalization or vectorization, and it expects text that has already be count vectorized.
+.. plot::
+    :context: close-figs
+    :alt: Frequency Distribution Plot
 
-We first instantiate a ``FreqDistVisualizer`` object, and then call ``fit()`` on that object with the count vectorized documents and the features (i.e. the words from the corpus), which computes the frequency distribution. The visualizer then plots a bar chart of the top 50 most frequent terms in the corpus, with the terms listed along the x-axis and frequency counts depicted at y-axis values. As with other Yellowbrick visualizers, when the user invokes ``poof()``, the finalized visualization is shown.
+    from sklearn.feature_extraction.text import CountVectorizer
+
+    from yellowbrick.text import FreqDistVisualizer
+    from yellowbrick.datasets import load_hobbies
 
-.. code:: python
+    # Load the text data
+    corpus = load_hobbies()
 
     vectorizer = CountVectorizer()
     docs       = vectorizer.fit_transform(corpus.data)
     features   = vectorizer.get_feature_names()
 
-    visualizer = FreqDistVisualizer(features=features)
+    visualizer = FreqDistVisualizer(features=features, orient='v')
```

Comment:
Great!  It is good to give the user some variety and also showcase different parameters.